src/components/Search/Search.css

```diff
@@ -6,8 +6,8 @@
   color: #ff3e3e;
 }
 
-.adoptButton,
-.adoptButton:hover {
+#adoptButton,
```

Comment:
why are we using ids here could we use className

Reply:
hi halit, yeah actually i had to. bcoz of the some duplicate classnames from other components, they override each other and break the css.